.gitignore

```diff
@@ -59,6 +59,7 @@ fns.txt
 .settings/
 *.sublime-workspace
 .vscode/
+.vmodules/
```

Comment:
that is not needed

Reply:
yes. it was for global thing. i forget to delete it

Reply:
.vscode/ should stay ignored - many people do use vscode for working on V;
.vmodules/ should not be ignored, and so should be deleted